arch/arm64/core/Kconfig

```diff
@@ -11,6 +11,15 @@ config CPU_CORTEX_A
 	help
 	  This option signifies the use of a CPU of the Cortex-A family.
 
+config CPU_CORTEX_R_AARCH64
+	bool
+	select CPU_CORTEX
+	select HAS_FLASH_LOAD_OFFSET
+	select USE_SWITCH
+	select USE_SWITCH_SUPPORTED
```

Comment:
Not needed anymore. Those should be implied by `CONFIG_ARM64` in `arch/Kconfig`

Reply:
OK, no problem